components/financial_assistance/app/domain/financial_assistance/operations/applications/pvc/submit_pvc_set.rb

```diff
@@ -76,6 +80,32 @@ def submit(params, family_ids)
             end
           end
 
+          def fetch_pvc_families_for_assistance_year(assistance_year, csr_list)
+            family_collection = Family.collection.aggregate([
```

Comment:
@vkghub A couple of things.
- Please make use of Grants. Review Family's Eligibility Determination.
- Move this to the Family class so this can be reused for other use cases.

Reply:
Good Catch, I will use grants.